docs/fiddles/features/represented-file/index.html

```diff
@@ -1,16 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-    <meta charset="UTF-8">
-    <title>Hello World!</title>
-    <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline';" />
-</head>
-<body>
-    <h1>Hello World!</h1>
-    <p>
-        We are using node <script>document.write(process.versions.node)</script>,
-        Chrome <script>document.write(process.versions.chrome)</script>,
-        and Electron <script>document.write(process.versions.electron)</script>.
-    </p>
-</body>
-</html>
```

Comment:
Just a thought: I'm finding it helpful to have the home page show basic directions about actions the user can take/expected changes. Just something like you have below:
>After launching the Electron application, click on the title with `Command` or `Control` key pressed. You should see a popup with the represented file at the top.